packages/compiler-cli/ngcc/BUILD.bazel

```
@@ -0,0 +1,14 @@
+load("//tools:defaults.bzl", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "ngcc",
+    srcs = glob([
+        "*.ts",
+    ]),
```

We can avoid the `glob` here:

```suggestion
    srcs = ["index.ts"],
```